dev-ostelco-ios-clientTests/CoordinatorTests/SingaporeUserHappyFlowWithScanICStageDeciderTests.swift

```diff
@@ -78,7 +78,45 @@ class SingaporeUserHappyFlowWithScanICStageDeciderTests: XCTestCase {
         
         XCTAssertEqual(decider.compute(context: context, localContext: localContext), .address)
     }
-    
+
+    func testUserHasCompletedJumioFailedCallback() {
```

Comment:
This name is unclear. What scenario are you testing?

Reply:
Forget the whole thing. I am removing this PR